.github/workflows/docs.yml

```diff
@@ -0,0 +1,58 @@
+name: Docs
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Actually we don't need to run this workflow in master, since the Antora builder runs outside of this repository.
Is there some kind of trigger that we need to execute on master? Or does it get updated regularly on a schedule without triggers?

Reply:
Afaik Antora runs based on a webhook that is configured in the k8up repo.

Reply:
It currently runs in a webhook. Could be improved by moving the CI trigger into a GitHub Action, like done for example here: https://github.com/appuio/appuio-io-docs/blob/master/.github/workflows/triggerci.yml. This gives better control.

But I don't think that it's urgent to change that now.

Reply:
Absolutely :+1: